dartagnan/src/test/resources/PTXv7_5-expected.csv

```diff
@@ -207,9 +207,29 @@ litmus/PTX/Manual/PC-bar-sync-sync-1.litmus,1
 litmus/PTX/Manual/PC-bar-sync-sync-2.litmus,1
 litmus/PTX/Manual/PC-bar-sync-sync-3.litmus,1
 litmus/PTX/Manual/PC-bar-sync-sync-4.litmus,1
+litmus/PTX/Memalloy/IRIW1.litmus,1
+litmus/PTX/Memalloy/IRIW2.litmus,1
 litmus/PTX/Memalloy/IRIW_gl_cta.litmus,1
 litmus/PTX/Memalloy/IRIW_uniproc.litmus,1
+litmus/PTX/Memalloy/IRRWIW1.litmus,1
+litmus/PTX/Memalloy/IRRWIW2.litmus,1
+litmus/PTX/Memalloy/IRRWIW3.litmus,1
+litmus/PTX/Memalloy/IRRWIW4.litmus,1
 litmus/PTX/Memalloy/IRRWIW_uniproc.litmus,1
+litmus/PTX/Memalloy/IRWIW1.litmus,0
+litmus/PTX/Memalloy/IRWIW2.litmus,0
 litmus/PTX/Memalloy/IRWIW_gl_cta.litmus,1
+litmus/PTX/Memalloy/RWC1.litmus,1
+litmus/PTX/Memalloy/RWC2.litmus,1
 litmus/PTX/Memalloy/RWC_uniproc.litmus,1
+litmus/PTX/Memalloy/WRC1.litmus,1
+litmus/PTX/Memalloy/WRC2.litmus,1
```

Comment:
All tests should also be added to `PTX6_0-exspected.csv`

Reply:
added